examples/ingest-data-generic-webhook.js

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
@liskaj Can you please name this file so that it better reflects it's purpose - using APS bearer token for authentication?

Reply:
@bakharal Renamed + updated comment on top of the file.